src/components/icons/NhlIcon.vue

```diff
@@ -1,3 +1,3 @@
 <template>
-  <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 16 16" aria-hidden="true" class="v-icon notranslate v-icon--is-component theme--light sport-icon"><path fill="#111" d="M9.5 14v1.5h-3V14h3zM15 .5l1 1-11 11H0l.004-2.75h5.75L15 .5zM9.747 9.25l.507.5h5.75L16 12.5h-5l-2.25-2.25.997-1zM1.132.64l6.114 6.114-.997.997L.135 1.638 1.132.64z"></path></svg>
+  <svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 16 16" aria-hidden="true" class="v-icon notranslate v-icon--is-component theme--light sport-icon"><path fill="#91969e" d="M9.5 14v1.5h-3V14h3zM15 .5l1 1-11 11H0l.004-2.75h5.75L15 .5zM9.747 9.25l.507.5h5.75L16 12.5h-5l-2.25-2.25.997-1zM1.132.64l6.114 6.114-.997.997L.135 1.638 1.132.64z"></path></svg>
```

Comment:
In SVGs you can also use `currentColor` and then whatever you set via css `color` will be inherited to the svg.

Reply:
Modified all SVGs to use currentColor